CRM/Contribute/BAO/Query.php

```diff
@@ -817,6 +817,7 @@ public static function defaultReturnProperties($mode, $includeCustomFields = TRU
         'fee_amount' => 1,
         // another
         'net_amount' => 1,
+        'tax_amount' => 1,
```

Comment:
Bang - there goes a kitten

Reply:
Because ```Contribution.get``` doesn't return tax_amount as default return properties :(

Reply:
OK - but is it possible for us to pass in a 'return' array instead of changing the defaults? That is the practice we encourage in general (ie. to usually pass in a return array when calling the api).

Also, I think this can go against the rc since so many related changes are in the rc